protocol/avdl/keybase1/teams.avdl

```diff
@@ -217,6 +222,7 @@ protocol teams {
     map<UserVersion, TeamBotSettings> restrictedBots;
     array<UserVersion> none;
     map<TeamInviteID,UserVersionPercentForm> completedInvites;
+    array<TeamUsedInvite> usedInvites;
```

Comment:
Good place for a comment about the difference between `completedInvites` and `usedInvites`. Like
```
// completedInvites is used for invites v8 and earlier.
// usedInvites is used for invites v9 and later. It's possible to 'use' the same invite multiple times in one link.
```

Reply:
what is a v8/v9? team chain version?